libs/designer/src/lib/core/DesignerProvider.tsx

```diff
@@ -35,6 +35,9 @@ export const DesignerProvider = ({ locale = 'en', options, children }: DesignerP
   const onError = useCallback<OnIntlErrorFn>((err) => {
     if (err.code === 'MISSING_TRANSLATION') {
```

Comment:
would we be able to replace this with our browser logger instead of a console log?
```
    if (err.code === 'MISSING_TRANSLATION' || err.code === 'MISSING_DATA') {
      LoggerService().log({
        level: LogEntryLevel.Warning,
        area: 'Designer Provider',
        message: err.message,
      });
      return;
    }
```

Reply:
Should it also include the code? ``message: `${err.code} - ${err.message}`,``

Reply:
Addressed both comments.